modules/common.nix

```diff
@@ -551,5 +576,6 @@ in {
     };
 
     passthru.inline = inline;
+    passthru.catalog = builtins.toJSON newCatalog;
```

Comment:
@limeytexan do you want to get this via a build so it becomes a file you can copy/read, or via a `nix eval --raw` in stdout?

Reply:
Ultimately we need access to the new catalog.json in order to compare it to the old one and calculate the adds/removes/upgrades. If the upgrade and import subcommands had an option to write the new floxenv to a temp target directory (in addition to the --no-render flag, without performing any generational link manipulation) then we could use that to perform the calculation.